src/getting-started/explanation.md

```
@@ -227,6 +234,208 @@ const result2 = await fn.expand(denops, "%");
+   {
+     "imports": {
+       "@denops/std": "jsr:@denops/std@^7.0.0"
+     }
+   }
+   ```
+
+### Why This Separation Matters
+
+When Vim/Neovim loads plugins, it merges all plugin directories into the runtime path. If runtime dependencies were in the root configuration file, they would conflict:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)